docs/conf.py

```diff
@@ -1,24 +1,32 @@
-"""
-Configuration file for the Sphinx documentation builder.
-"""
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
 
 import os
 import datetime
 from pathlib import Path
 
-from sunpy_sphinx_theme import PNG_ICON
+from sunpy_sphinx_theme import *
```

Comment:
```suggestion
from sunpy_sphinx_theme import PNG_ICON
```

Reply:
in my experience so far, this doesn't work, but I may have been doing it wrong

Reply:
Maybe you have an older version of the theme? That should work and if it doesn't, we need to fix it in the theme.